plugins/leemons-plugin-users/frontend/src/components/SelectProfile.js

```diff
@@ -32,7 +30,7 @@ const SelectProfile = forwardRef(({ firstSelected, onChange, ...props }, ref) =>
         value: id,
       }));
 
-      if (firstSelected) {
+      if (firstSelected && store.data.length > 0) {
```

Comment:
`firstSelected && store.data.length > 0` is the same as `firstSelected && store.data.length`